index.html

```diff
@@ -117,21 +117,31 @@ <h2>API</h2>
 import * as parsel from "./parsel.js";
 self.parsel = parsel; // so that people can experiment in the console
 
+if(location.search.includes('?selector=')) {
+
+	let selector = decodeURIComponent(location.href.split('?selector=')[1])
+	selectorText.value = selector || selectorText.value // set default value if selector is empty
+	
+	history.replaceState(null, '', `?selector=${selector}`)
+}
+
 (tryout.oninput = e => {
-	tokens.textContent = JSON.stringify(parsel.tokenize(selectorText.value), null, "\t");
-	Prism.highlightElement(tokens);
-
-	try {
-
-		tree.textContent = JSON.stringify(parsel.parse(selectorText.value, {recursive: recursive.checked}), null, "\t");
-		tree.classList.remove("error");
-		specificity.textContent = parsel.specificity(selectorText.value);
-		Prism.highlightElement(tree);
-	}
-	catch (e) {
-		tree.classList.add("error");
-		tree.innerHTML = `<details><summary>${e}</summary>${e.stack.replace(e, "")}</details>`;
-	}
+history.replaceState(null, '', `?selector=${selectorText.value}`)
+	
+tokens.textContent = JSON.stringify(parsel.tokenize(selectorText.value), null, "\t");
+Prism.highlightElement(tokens);
+
+try {
+
+	tree.textContent = JSON.stringify(parsel.parse(selectorText.value, {recursive: recursive.checked}), null, "\t");
+	tree.classList.remove("error");
+	specificity.textContent = parsel.specificity(selectorText.value);
+	Prism.highlightElement(tree);
+}
+catch (e) {
+	tree.classList.add("error");
+	tree.innerHTML = `<details><summary>${e}</summary>${e.stack.replace(e, "")}</details>`;
+}
```

Comment:
Why are all these lines in the diff even though they haven't been changed? It looks like there was some whitespace edit? Perhaps removing indentation? Could you please change it back?

Reply:
> Hi there @umutbozdag, are you still interested in working on this? I need the functionality so if not, I'll implement it myself, but I don't want to step on your toes if you still want to work on it.

Sorry, i was too busy. I will probably finish this today. Thanks for pinging :)